phpunit.xml.dist

```diff
@@ -20,7 +20,7 @@
         convertWarningsToExceptions="true"
         processIsolation="false"
         stopOnFailure="false"
-        bootstrap="tests/_bootstrap.php"
+        bootstrap="vendor/autoload.php"
```

Comment:
bootstrap should boostrap.php which is vendor/autoload.php includes

Reply:
It's just simplification since he removed _bootstrap.php. It didn't doanything other than include autoload.php.